internal/spec/protection.go

```diff
@@ -6,10 +6,25 @@ import (
 
 // Protection rule of repositry branch
 type Protection struct {
-	// Require status checks for merge
-	Require []string `yaml:"require,omitempty"`
+	// Required status checks for merge
+	Checks []string `yaml:"checks,omitempty"`
```

Comment:
Documentation comment line should start with the same word as field name (`Checks` in this case) to generate proper documentation